doctor/utils/__init__.py

```diff
@@ -3,6 +3,8 @@
 import re
 import sys
 
+import six
+from past.builtins import execfile
```

Comment:
I guess we could always just define our own execfile in here if we can't get mocking working on exec. (Rather than adding future as a dependency just for that.)